src/mesh/meshblock.cpp

```diff
@@ -179,25 +179,25 @@ void MeshBlock::Initialize(int igid, int ilid, LogicalLocation iloc,
   // potentially used in the load balancing calculation, but not all
   // variables that we may want to communicate are necessarily relevant
   // to the cost per meshblock.
-  const auto vars = real_container
-                        ->GetVariablesByFlag({Metadata::Independent, Metadata::FillGhost,
-                                              Metadata::ForceRemeshComm},
-                                             false)
-                        .vars();
-  for (int n = 0; n < vars.size(); ++n) {
-    RegisterMeshBlockData(vars[n]);
+  using FS_t = Metadata::FlagCollection;
```

Comment:
Why `FS_t` and not `FC_t`?

Reply:
Artifact from an earlier draft when this type was called `FlagSet` not `FlagCollection`. Fixed.